docs_manual/source/ext_tool_cyberduck.rst

```diff
@@ -0,0 +1,56 @@
+.. _ext_tool_cyberduck:
+
+Cyberduck
+^^^^^^^^^
+
+Cyberduck is a libre network storage browser for Mac and Windows.
+It can be used to access the iRODS filesystem on SODAR directly.
```

Comment:
We should clarify our position a bit on Cyberduck. It's not really "officially" supported by us, but it can be a useful tool worth trying out. Maybe write something like that?

Reply:
Will need to take new screenshots as well.